js/clinicalTimeline.js

```diff
@@ -702,8 +702,11 @@ window.clinicalTimeline = (function(){
           $(a).on("click", hideTrackClickHandler(elem.prop("innerHTML")));
           $(trackTooltip).append(a);
           $(trackTooltip).append("<br />");
-
-          if (!isDurationTrack(getTrack(allData, elem.prop("innerHTML")))) {
+		  
+		var trackData = getTrack(allData, elem.prop("innerHTML"));
+		var d3timeline = d3.timeline();
+		d3timeline.groupCount(trackData.times, stackSlack);
+		if ((!isDurationTrack(trackData)) && (d3timeline.groupCount() !=  trackData.times.length)) { 
```

Comment:
The last conditional `(d3timeline.groupCount() !=  trackData.times.length)` should only apply to the Collapse/Stack button, not to the other options such as color by and split by.